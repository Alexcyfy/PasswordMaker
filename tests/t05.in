123456789
a
123456789

