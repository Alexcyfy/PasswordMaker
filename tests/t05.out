Password: 
Backup password: Password must be at least 8 characters long.

Thank you.
